wsagent/che-core-api-project/src/main/java/org/eclipse/che/api/project/server/impl/ExecutiveProjectManager.java

```diff
@@ -295,8 +284,9 @@ public synchronized RegisteredProject removeType(String wsPath, String type)
       throws ConflictException, NotFoundException, ServerException, BadRequestException,
           ForbiddenException {
 
-    RegisteredProject project =
-        get(wsPath).orElseThrow(() -> new NotFoundException("Can't find project"));
+    RegisteredProject project = projectConfigRegistry.getOrNull(wsPath);
```

Comment:
I guess we can simply make it either 
```java
RegisteredProject project = projectConfigRegistry.get(wsPath).orElseThrow(() -> new NotFoundException("Can't find project"));
``` 
or
```java
RegisteredProject project = projectConfigRegistry.getOrNull(wsPath);
if (project == null) {
   throw new NotFoundException("Can't find project");
}
```

Reply:
done